docs/auto-discovery.md

```diff
@@ -47,11 +47,23 @@ are using.  The following three variables are common to all observers:
 
 For a list of observers and the discovery rule variables they provide, see [Observers](./observer-config.md). 
 
+### <a name="additionalfns"></a>Additional Functions
```

Comment:
Take out the `<a />` tag and use `additional-functions`, which is autogenerated by github and pandoc.